Desktop/widgets/listmodelfactorsform.cpp

```diff
@@ -78,7 +78,16 @@ void ListModelFactorsForm::initFactors(const FactorVec &factors)
 	}
 	
 	_setTerms(newTerms);
-	endResetModel();	
+	endResetModel();
+}
+
+int ListModelFactorsForm::countVariables() const
+{
+	int count = 0;
```

Comment:
You can probably use an `std::accumulate` instead.

Reply:
Yes but that is not my code ;)